src/components/shared/thread/ActivityGraph.js

```diff
@@ -4,12 +4,10 @@
 // @flow
 
 import * as React from 'react';
-import { computeActivityGraphFills } from './ActivityGraphFills';
-import { timeCode } from '../../../utils/time-code';
+import { ActivityGraphCanvas } from './ActivityGraphCanvas';
 import classNames from 'classnames';
 import { Tooltip, MOUSE_OFFSET } from '../../tooltip/Tooltip';
 import SampleTooltipContents from '../SampleTooltipContents';
-import { mapCategoryColorNameToStyles } from '../../../utils/colors';
```

Comment:
This change to named export should have been done in my onboarding task, I missed the file so I completed the task here.